go/vt/vtgate/planbuilder/operators/operator.go

```diff
@@ -89,6 +89,21 @@ func PlanQuery(ctx *plancontext.PlanningContext, stmt sqlparser.Statement) (ops.
 		return nil, ctx.SemTable.NotSingleRouteErr
 	}
 
+	// set lock and comments on the route to be set on the sql query on conversion.
```

Comment:
this should be extracted into it's own method to follow the rest of `PlanQuery` flow